packages/dev/loaders/src/SPLAT/splatLoadingOptions.ts

```diff
@@ -6,4 +6,8 @@ export type SPLATLoadingOptions = {
      * Defines if buffers should be kept in memory for editing purposes
      */
     keepInRam?: boolean;
+    /**
+     * Spatial Y Flip for splat position and orientation
+     */
+    flipY?: boolean;

```

Comment:
Should we add this to `_DefaultLoadingOptions` for consistency?

Reply:
yes! Fixed.